lib/reverse_words.rb

```diff
@@ -84,9 +84,9 @@ def reverse_words(my_words)
     reversed_string = string_reverse(my_words, start_index, end_index)
```

Comment:
Since all your manipulations are happening in place, you don't need to return my_string on line 58 and set it to reversed_string on line 84. Also, you don't need to return reversed_string on line 88. Line 58 and line 88 could simply be changed to `reverse`.